docs/extending.md

```diff
@@ -12,13 +12,19 @@ Currently, Narwhals has full API support for the following libraries:
 | Modin | [github.com/modin-project/modin](https://github.com/modin-project/modin) |
 | PyArrow ⇶ | [arrow.apache.org/docs/python](https://arrow.apache.org/docs/python/index.html) |
 
-It also has lazy-only support for [Dask](https://github.com/dask/dask), and interchange-only support
+It also has **lazy-only** support for [Dask](https://github.com/dask/dask), and **interchange** support
 for [DuckDB](https://github.com/duckdb/duckdb) and [Ibis](https://github.com/ibis-project/ibis).
 
 ### Levels of support
 
-Narwhals comes with two levels of support ("full" and "interchange"), and we are working on defining
-a "lazy-only" level too.
+Narwhals comes with three levels of support:
+
+- **Full API support**: cuDF, Modin, pandas, Polars, PyArrow
+- **Lazy-only support**: Dask
+- **Interchange-level support**: DuckDB, Ibis, Vaex, anything which implements the DataFrame Interchange Protocol
+
+The lazy-only layer is a major item on our 2025 roadmap, and hope to be able to bring libraries currently in
+the "interchange" level into that one.
```

Comment:
we already had all 3 levels on the README, so I think we should mention all 3 here too